worker/src/RTC/TransportCongestionControlClient.cpp

```diff
@@ -244,35 +290,52 @@ namespace RTC
 		// more stable values.
 		this->bitrates.startBitrate = std::max<uint32_t>(MinBitrate, this->bitrates.availableBitrate);
 
+		auto prevMaxBitrate = this->bitrates.maxBitrate;
+
 		if (this->desiredBitrateTrend.GetValue() > 0u)
 		{
 			this->bitrates.maxBitrate = std::max<uint32_t>(
```

Comment:
I'd prefer to do is not to update `this->bitrate.maxBitrate` unless it actually changes (namely use local variable and only update property when needed)